### DISCUSSION TITLE
"Name" of subproject should be visible for all users, but not "projected budget"

### DISCUSSION THREAD

**Discussion by SophieKettler** [openkfw/TruBudget#856] [General]
It is only possible that a user sees the entire subproject including the projected budget or does not see it at all. A mix of "see subproject, but not the projected budget" does not work. However, it's crucial for the use of TruBudget as f.ex. consultant A should not see the contract amount (projected budget) of consultant B but the "subproject (name".

**Comment by Stezido** [comment]
I see, so the projected-budget of a project/subproject/ and the assigned/disbursed budget of a workflowitem should have an own permission which can be granted/revoked using the permission endpoints. This is a valid request and requires a lot of work. Due to the effort and changes on api level I suggest to postpone it to v2.0.0
